### ISSUE TITLE
How can you train this model on your own data?

### ISSUE THREAD

**Issue by s724959099** [miranthajayatilake/YOLOw-Keras#1]
Hi, I am a novice.
Can you give a mini-tutorial on how to train this model and use it to predict bounding-boxes, on your own data.
I don't know how to get started it.

**Comment by miranthajayatilake** [comment]
Hi, here are the steps!
1. Clone this repository to your PC
2. Download any Darknet model cfg and weights from the official YOLO website.
3. Convert the downloaded cfg and weights files into a h5 file using YAD2K library. 
4. Copy the generated h5 file to the model_data folder and edit the name of the pretrained model in yolo.py code to the name of your h5 file.
5. Place the input image you want to try object detection in the images folder and copy its file name.
6. Assign your input image file name to input_image_name variable in yolo.py.
7. Open terminal from the repository directory directly and run the yolo.py file
`python yolo.py`

**Comment by s724959099** [comment]
I've started a quick start before write to here.
Where is the training part in yolo.py?
Or it just predict boxes?
Thank you for your answer.

**Comment by andikira** [comment]
Hello @miranthajayatilake  i like this repo, you 're doing well
i'm a beginner in deep learning and computer vision. 
I have same issues like @s724959099, how to train new class with our images and our bounding box label with pre-train weight ? 
i think in the steps you've write above is step to detect object in coco class of new image 

Thanks in advance

**Comment by cxf2015** [comment]
@s724959099 @andikira  https://github.com/allanzelener/YAD2K

**Comment by andikira** [comment]
thanks @cxf2015 for your answer. I'll check that repo

**Comment by Antanskas** [comment]
@miranthajayatilake 
Can You please guide me how can I train maybe already pre-trained model for even one object class to detect. To aim is just practice to train Yolo model by myself. I am interested in all the steps required:
prepare training data
load pre-trained mode
feed training data to model
test new model on test data
p.s. i would like to use keras for that
